extension/background.js

```diff
@@ -65,6 +65,7 @@ chrome.browserAction.onClicked.addListener(function (tab) {
             "lib/tracker.js",
             "lib/display.js",
             "lib/doc.js",
+            "lib/ui.js",
```

Comment:
so from seeing [display.js, ui.js], im not sure what either of them does. they just sound like synonyms to me.
maybe:
- time_tracker_view.js
- settings_view.js

The _view suffix is because someday if we have time i might want to do MVC: https://www.guru99.com/mvc-tutorial.html#:~:text=The%20Model%2DView%2DController%20(,development%20aspect%20of%20an%20application.
Then you would have settings_view, settings_model, settings_controller

Reply:
can be done in a future cl too.
when you want to change file names its good to have a CL that really is just changing names, and no other change.

Reply:
Agree